srcpkgs/vapoursynth-editor/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'vapoursynth-editor'
 pkgname=vapoursynth-editor
-version=r18
+version=19
 revision=1
-_bitbucket_hash=a511db5a522c
+_bitbucket_hash=8019bb5a8099
 wrksrc="mystery_keeper-${pkgname}-${_bitbucket_hash}"
 build_style=qmake
 hostmakedepends="qt5-qmake"
 makedepends="vapoursynth-devel qt5-devel qt5-websockets-devel"
 depends="vapoursynth"
-short_desc="A cross-platform editor for VapourSynth script"
+short_desc="Cross-platform editor for VapourSynth script"
 maintainer="DirectorX <void.directorx@protonmail.com>"
 license="MIT"
 homepage="https://bitbucket.org/mystery_keeper/vapoursynth-editor"
-distfiles="${homepage}/get/${version}.tar.gz"
-checksum=7c560c1ef4ea1810babda30f11358e073ba471971a43a0252bc0ae04d3de901f
+distfiles="${homepage}/get/r${version}.tar.gz"
+checksum=41e09a6c1411536f7a223a16263145b31de4715189cfe2ebe62b4ad69d6ec342
+nocross="vapoursynth is nocross"
```

Comment:
This is extremely redundant, please provide a short version of the error that happens when trying to cross compile